src/models/components/controller.py

```diff
@@ -52,8 +53,18 @@ def __init__(
             hidden_size=c_hidden_size,
             bias=bias,
         )
-        self.fc_mean = torch.nn.Sequential(torch.nn.Linear(c_hidden_size, action_size))
-        self.fc_logs = torch.nn.Sequential(torch.nn.Linear(c_hidden_size, action_size))
+        self.fc_mean = LinearLayers(
+            input_size=c_hidden_size,
+            hidden_size=c_hidden_size * 2,
+            layers=3,
+            output_size=action_size,
+        )
+        self.fc_logs = LinearLayers(
+            input_size=c_hidden_size,
+            hidden_size=c_hidden_size * 2,
+            layers=3,
+            output_size=action_size,
+        )
```

Comment:
fc_meanとfc_logsは計算量の削減のためにただの線形層でお願いします
その前に`fc_hiddens`等の名前で`LinearLayer`を定義しておいて、そこから分岐させる方法を取りましょう。

LinearLayersのhidden_sizeもコンストラクタの引数から直接指定できるようにしましょう。レイヤーの数も同様です。
```suggestion
        self.fc_hiddens = LinearLayers(c_hidden_size, stacked_linear_hidden_size)
        
        self.fc_mean = nn.Linear(stacked_linear_hidden_size, action_size)
        self.fc_logs = nn.Linear(stacked_linear_hidden_size, action_size)
```